messages/maintainer_access.md

```diff
@@ -0,0 +1,19 @@
+Hello {NAME},
+
```

Comment:
Needs an intend at the top. This is one possible place that new maintainers might come to when they want to know how they become a maintainer, who can nominate them, if they can self-nominate. As discussed in the Coco, this is not the place we intend to lay this out. So, this needs a statement of intend and a point to where to find that info on the top.

Reply:
should be in the commit I'm about to push up